docs/update.md

```diff
@@ -35,6 +35,9 @@ This is a detailed documentation for the update functionality of packit.
     2. `export PAGURE_FORK_TOKEN=<token>` — packit needs this token to create a
        pull request:
        https://src.fedoraproject.org/fork/YOU/rpms/PACKAGE/settings#apikeys-tab
+    3. `export PAGURE_PACKAGE_TOKEN=<token>` - packit needs this token to access
```

Comment:
the truth is that we don't need this token, yet; I would prefer to change packit so that it would not require it so far

Reply:
I'll change it.

Reply:
https://github.com/packit-service/packit/pull/81/commits/cbb2bca1195b5b1276bc5aff8d927c9e8f78887e